### ISSUE TITLE
Negative TimeZone

### ISSUE THREAD

**Issue by ITBlandMCT** [p1ne/fdim-controller#1]
Time from GPS is correct only before 09:59, needs bcdToDec() applied or it will be 4 hours ahead for the rest of the day, when using 12Hr clock. (Line 360 of can-clock.ino)

**Comment by ITBlandMCT** [comment]
No, something else wrong. Still looking at it (I am getting +4 hours for my timezone (-5))

**Comment by p1ne** [comment]
Time from GPS is in UTC (like GMT or BST but without daylight/no daylight considered). So just try to set TZ parameter to -4 (if you're in -5, it could be -5 difference but +1 daylight saving)

**Comment by ITBlandMCT** [comment]
my TZ is set to -5, the time shows incorrectly (+4 from my local time)--I am in US CDT which is normally -6, but is -5 for the summer. 
The TZ value is uint8_t so is the -5 being converted to +5?

**Comment by ITBlandMCT** [comment]
OK, I changed the tz value to an **int8_t** and that seems to have fixed the problem
Edit: moved the rest of this comment to the next issue

**Comment by ITBlandMCT** [comment]
EDIT: Moved to new issue.

**Comment by ITBlandMCT** [comment]
Simplifying issues list, dedicating this issue to just the TimeZone issue:

fix appears to be changing tz value to an int8_t (was unit8_t), unless there is a better way to handle it.

Note: 12 Hr problem moved to new issue
